feature-group-definitions/container-queries.yml

```diff
@@ -4,5 +4,17 @@ usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/4165
 status:
   is_baseline: false
   support:
-    chrome: "105"
-    safari: "16"
+    chrome: "111"
+    edge: "111"
+    firefox: "110"
+compat_features:
+  - api.CSSContainerRule
+  - api.CSSContainerRule.containerName
+  - api.CSSContainerRule.containerQuery
+  - css.at-rules.container
+  - css.properties.contain
+  - css.properties.contain.inline-size
+  - css.properties.contain.style
+  - css.properties.container
+  - css.properties.container-name
+  - css.properties.container-type
```

Comment:
I don't know if this feature list is good, to be honest. Safari is missing the CSSOM bit. If we relaxed this requirement, then we'd be looking at Safari 17 as the baseline-effective version.

Reply:
I think it's fine to not consider CSSOM part of CQ, it matters very little in practice.

Reply:
OK, in that case this is a coming-soon feature, which we don't want to show just yet. I'll drop the CSSOM bits from the feature list but update-and-comment-out the baseline data.